src/PlusVolumeReconstruction/vtkPlusVolumeReconstructor.cxx

```diff
@@ -438,8 +438,7 @@ PlusStatus vtkPlusVolumeReconstructor::SetOutputExtentFromFrameList(vtkPlusTrack
   for (int d = 0; d < 3; d++)
   {
     // in general, this would be computed as int(std::floor((extent_Ref[d * 2] - outputOrigin_Ref[d]) / outputSpacing[d]));
-    // we set outputOrigin_Ref so that this will be always 0
-    outputExtent[d * 2] = 0;
+    // we set outputExtent so that this will be always 0

```

Comment:
Could you provide a link to the first place this is initialized to 0?

Reply:
A few lines above, line 437 perhaps.